README.md

```diff
@@ -1,7 +1,37 @@
 # MafiaScum Forum
 This repo hosts the code used on https://forum.mafiascum.net
 
-# With Docker
+# With Docker Compose
+This assumes you already have both Docker and Docker Compose installed.
+
+## Setup
+Copy `.env.sample` to `.env` and modify accordingly if this suits you.
```

Comment:
Probably should add `.env` to the gitignore.

Reply:
done.